1773.count-items-matching-a-rule.js

```diff
@@ -0,0 +1,35 @@
+/*
+ * @param {string[][]} items
+ * @param {string} ruleKey
+ * @param {string} ruleValue
+ * @return {number}
+ */
+
+/* URL of this problem
+    https://leetcode.com/problems/count-items-matching-a-rule/
+*/
+
+var countMatches = function(items, ruleKey, ruleValue) {
+    let ruleKeyIndex;
+    switch (ruleKey) {
+        case "type":
+            ruleKeyIndex = 0;
+            break;
+        case "color":
+            ruleKeyIndex = 1;
+            break;
+        case "name":
+            ruleKeyIndex = 2;
+            break;
+    }
+    
+    let count = 0;
```

Comment:
the variable name is not self-descriptive.

Reply:
I change the name of the variable to matchingItemCount.
Is this name self-descriptive enough?